lib/MusicBrainz/Server/Form/Filter/Recording.pm

```diff
@@ -44,12 +48,20 @@ sub options_video {
     ];
 }
 
+sub options_works {
+    return [
+        { value => 1, label => l('Linked to works') },
+        { value => 2, label => l('Not linked to works') },
```

Comment:
I was wondering if we should use "Related to works" instead (the name of the button in the relationship editor is "Add related work," for example), though we are probably inconsistent about that.

Reply:
Fair point, made the change.